crypto/rand/rand_unix.c

```diff
@@ -64,7 +64,7 @@ static uint64_t get_timer_bits(void);
  * macro that might be undefined.
  */
 # undef OSSL_POSIX_TIMER_OKAY
-# if defined(_POSIX_TIMERS) && _POSIX_TIMERS > 0
+# if defined(_POSIX_TIMERS) && _POSIX_TIMERS > 0 && !defined(__sun)
```

Comment:
To see the history of the check for `OSSL_POSIX_TIMER_OKAY`, annotate `rand_lib.c`, not `rand_unix.c`. Although git attributes these lines here to me, I only moved them here in 5bc6bcf82d2a.

2b66fd5720c3 Unify timer code (#5231)
bed4afa81b9c Fix glibc specific conditional for Mac OS/X (#5269)
4cd58771d838 Fix glibc version detection. (#5279)